docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -0,0 +1,373 @@
+
+
+### Method C
+For definition see [Method C](jpsreport_method_C.html). Method C is used to analyze the steady state.
+
+```xml
+  <method_C enabled="true">
+      <measurement_area id="1"/>
+  </method_C>
+```
+
+Possible parameters are:
+
+- `id` indicates the size and location of the measurement_area.
+   Several measurement areas can be set in one inifile.
+
+- up to version 0.8.5:  `plot_time_series` specifies whether output the ($$\rho-t$$) and ($$v-t$$) diagrams.
+
+
+Possible output data are:
+  - `/Fundamental_Diagram/Classical_Voronoi/`: output file `rho_v_Classic_traj_` with mean density and velocity of over time (frame, $$rho(t)$$, $$v(t)$$).
+
+
+### Method D
+For definition see [Method D](jpsreport_method_D.html). Method D is used to analyze the steady state.
+
+```xml
+ <method_D enabled="true">
+   <measurement_area id="1" start_frame="None" stop_frame="None"
+         get_individual_FD="false"/>
+   <one_dimensional enabled="false"/>
+   <cut_by_circle enabled="false" radius="1.0" edges="10"/>
+   <profiles enabled="false" grid_size_x="0.20" grid_size_y="0.20"/>
+ </method_D>
+```
+
+Possible parameters are:
+
+- For each `measurement_area`, several id numbers can be set in one inifile.
+  `start_frame` and `stop_frame` give the starting and ending frame for data analysis.
+  The default values of these two parameters are `None`.
+  If you plan to analysis the whole run from beginning to the end,
+  set both of `start_frame` and `stop_frame` as `None`;
+  If `start_frame =None` but `stop_frame` is not,
+  then analysis will be performed from beginning of the trajectory to the `stop_frame`.
+  If `start_frame` is not `None` but `stop_frame = None`,
+  it will analyze from the `start_frame` to the end of the movement.
+  `get_individual_FD` determines whether or not to output the data
+  for individual fundamental diagram in the given measurement area,
+  which is based on the Voronoi density $$\rho_i$$,  velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon
+  of each pedestrian $$i$$ in a given measurement area but not mean
+  value over space.
+  If true, the related data will be written in the
+  folder `./Output/Fundamental_Diagram/IndividualFD/` in the output file `IFD_D_`.
+
+ - up to version 0.8.5: `plot_time_series` specifies whether output the $$\rho-t$$ and $$v-t$$-diagram.
+
+- `one_dimensional` should be used when pedestrians move on a line
+  [single-file experiment](http://www.asim.uni-wuppertal.de/datenbank/own-experiments/corridor/1d-single-file-no-2.html).
+
+- `cut_by_circle` determines whether to cut each cell by circle or not.
+         Two options `radius` of the circle and the number of `edges` have
+         to be supplied for approximating the circle if `enabled` is *true*.
+
+- up to version 0.8.5: `output_voronoi_cells` specifies whether or not to output data for visualizing
```

Comment:
The version number must be corrected.

Reply:
here again, good question. After the union of jpscore and jpsreport, @zeroset and you (@chraibi) should clarify this. JPScore is currently at 0.8.4. JPSreport was already at 0.8.5.

Reply:
see [suggestion](https://github.com/JuPedSim/jpscore/pull/626#discussion_r360882255)